pwncat/platform/windows.py

```diff
@@ -635,7 +635,7 @@ def _bootstrap_stage_two(self):
             "\\Windows\\System32\\spool\\SERVERS",
             "\\Windows\\System32\\spool\\drivers\\color",
             "\\Windows\\System32\\Tasks\\Microsoft\\Windows\\SyncCenter",
-            "\\Windows\\System32\\Tasks_Migrated (after peforming a version upgrade of Windows 10)",
+            "\\Windows\\System32\\Tasks_Migrated (after performing a version upgrade of Windows 10)",
```

Comment:
Wait, what is this?
Was this supposed to be a __comment__?

Reply:
I removed that string. You're right. I think that made it in by accident.